doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -263,8 +263,9 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
      - Open Virtual Machine Firmware (OVMF) is an EDK II based project to enable
        UEFI support for Virtual Machines.
 
-       ACRN currently does not support off-the-shelf OVMF builds targeted for
-       QEMU and KVM. This feature is still under development.
+       ACRN does not support off-the-shelf OVMF builds targeted for QEMU and
```

Comment:
@hwang37 @NanlinXie Do we need to document the files in this folder, or should this PR be merged as is?

Reply:
There is a `changelog` file in that folder that provides some history pertaining to the vSBL boot firmware. I think a few enhancements would be nice:
1. Add a changelog for the OVMF boot firmware
2. Add the `md5sum` info for *all* boot firmware in the `MD5SUM` file (not just the vSBL)
3. Use `sha512sum` instead of `md5sum`

While these are nice to have, I don't feel they should block merging this PR. My 2 cents...

Reply:
@gvancuts Thanks for your suggestions. I can definitely add those later in a different PR.

Reply:
Addressed in https://github.com/projectacrn/acrn-hypervisor/pull/2869